pkg/utils/initialPatchEngine.go

```diff
@@ -3,28 +3,30 @@ package utils
 import (
 	log "github.com/sirupsen/logrus"
 
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-
 	"github.com/litmuschaos/chaos-operator/pkg/apis/litmuschaos/v1alpha1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+type ExperimentStatus v1alpha1.ExperimentStatuses
+
+// IntialExperimentStatus fills up ExperimentStatus Structure with intialValues
+func (expStatus *ExperimentStatus) IntialExperimentStatus(experimentDetails *ExperimentDetails) {
+	expStatus.Name = experimentDetails.JobName
+	expStatus.Status = "Waiting for Job Creation"
+	expStatus.Verdict = "Waiting"
```

Comment:
instead of `waiting`, `pending` will be good

Reply:
I think waiting, would be good.